toolkit/offchain/src/init_governance/mod.rs

```diff
@@ -169,3 +164,27 @@ pub async fn get_governance_utxo<T: QueryLedgerState + Transactions + QueryNetwo
 
 	Ok(governance_utxo)
 }
+
+pub(crate) struct GovernanceData {
+	policy_script: PlutusScript,
+	utxo_id: UtxoId,
+}
+
+impl GovernanceData {
+	pub(crate) fn policy_script_hash(&self) -> ScriptHash {
+		self.policy_script.hash()
+	}
+}
+
+pub(crate) async fn get_governance_data<T: QueryLedgerState + Transactions + QueryNetwork>(
+	genesis_utxo: UtxoId,
+	client: &T,
+) -> anyhow::Result<GovernanceData> {
+	let utxo = get_governance_utxo(genesis_utxo, client).await?;
+	let utxo_id = utxo.to_domain();
+	if let Some(OgmiosScript::Plutus(ps)) = utxo.script.clone() {
+		Ok(GovernanceData { policy_script: PlutusScript::new_v2(ps.cbor), utxo_id })
+	} else {
+		Err(anyhow!("Governance UTXO script is not PlutusScript"))
+	}
+}
```

Comment:
This is unused now, but I'll use it in `init-reserve-management` (PR in progress) and I think D parameter and permissioned candidates could use it